vitality/templates/login.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>login</title>
+	<link rel = "stylesheet" type = "text/css" href = "{{url_for("static", filename="login.css")}}">
+
+</head>
+
+<body>
+	<header>
+  		<nav class="navd">                                      <!--Navagation Bar-->
+    		<ul>
+      			<li><a href="index.html">Log In</a></li>
+     			<li><a href="createuser.html">Creat Account</a></li>
+    		</ul>
+  		</nav>
+  	 	  
+	</header>
+<h2>Login Form</h2>
+<form action="/action_page" method="post">
+  <div class="image"> <img src="men-img2.png" alt="Avatar" class="avatar"> </div>
+  <div class="container">
+    <label for="uname"><b>Username</b></label>
+    <input type="text" placeholder="Enter Username" name="uname" required>
+    <label for="psw"><b>Password</b></label>
+    <input type="password" placeholder="Enter Password" name="psw" required>
+    <button type="submit">Login</button>
+    <label>
+      <input type="checkbox" checked="checked" name="remember">
+      Remember me </label>
+  </div>
+  <div class="container" style="background-color:#f1f1f1">
+    <button type="button" class="cancelbtn">Cancel</button>
+    <span class="psw">Forgot <a href="#">password?</a></span> </div>
+  <div class="container" style="background-color:#f1f1f1">
+    
+  </div>
+</form>
+<footer class="Validation">                                              <!--Validate HTML and CSS--> 
+	<a href="https://validator.w3.org/check?uri=referer">Validate HTML</a> 
+	<a href="https://jigsaw.w3.org/css-validator/check/referer">Validate CSS</a>
+</footer>
```

Comment:
Will need to remove these validation lines since jninja will not work in validator.w3.org. 

Don't worry for now. Going to approve to merge files for now. Then will need these changed.